app/src/main/res/layout/content_no_internet.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout
+        xmlns:android="http://schemas.android.com/apk/res/android"
+        xmlns:tools="http://schemas.android.com/tools"
+        xmlns:app="http://schemas.android.com/apk/res-auto" >
+
+    <data>
+        <variable name="title" type="com.androidtitan.culturedapp.model.newyorktimes.Article" />
+    </data>
+
+    <android.support.v4.widget.NestedScrollView
+            app:layout_behavior="@string/appbar_scrolling_view_behavior"
+            tools:showIn="@layout/no_internet_activity"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            tools:context="com.androidtitan.culturedapp.main.offline.NoInternetActivity">
+
+        <TextView
+                android:id="@+id/title_textView"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:padding="@dimen/text_margin"
+                android:gravity="center"
+                android:background="@color/colorPrimary"
+                android:textColor="@android:color/white"
+                android:text="@{title.title}"/>
```

Comment:
Expressions within the layout are written in the attribute properties using the `"@{}"` syntax.